source/rfc/external.rst

```diff
@@ -71,6 +74,10 @@ Location:
 
 * https://files.pythonhosted.org (package files)
 
+Configuration required:
```

Comment:
Change this to "mirror consumption instructions" or similar. Right now it's not clear what is being configured (mirror or the clients).